### DISCUSSION TITLE
How about add "isEmpty()" to vars ?

### DISCUSSION THREAD

**Discussion by yellowbean** [reflex-dev/reflex#3080] [Ideas]
for vars with type of `dict` or `list` 

I usually to create extra variable to serve as flag to display a component or not .

  class FormState(rx.State):
  
      messages: list[str]
      user: dict[str, str]

      @rx.var
      def isMessagesEmpty() -> bool :
          return len(self.messages)==0

      @rx.var
      def isUserEmpty() -> bool :
          return len(self.user.keys())==0

Then in the UI component..

    rx.cond(State.isUserEmpty ....)

Would it be better off to save noise code by adding a method like `isEmpty()` to the vars ?

    class FormState(rx.State):
    
        messages: list[str]
        user: dict[str, str]

Then in the UI component..

    rx.cond(State.user.isEmpty() ....)

**Comment by masenf** [comment]
If you just use the list/dict var directly in the cond does it not work?
